src/renderer/i18n/locales/it-IT.js

```diff
@@ -0,0 +1,964 @@
+    INVALID_URI: 'URI non valido',
+    INVALID_FORMAT: 'Formato non valido',
+    MAX_LENGTH: 'Max {0}',
+    MUST_BE_GREATER_THAN: 'Deve essere maggiore di {0}',
+    NOT_MATCH: '\'{0}\' non coincide con \'{1}\'',
+    NOT_VALID: '\'{0}\' non è valido',
+    NOT_NUMERIC: '\'{0}\' non è numerico',
+    NO_SCHEME: '\'{0}\' non contiene \'http://\' o \'https://\'',
+    NAME: {
+      DUPLICATED: 'Il nome \'{0}\' esiste già',
+      EXISTS_AS_CONTACT: 'Il nome \'{0}\' è già stato assegnato a un contatto',
+      EXISTS_AS_WALLET: 'Il nome \'{0}\'è già stato assegnato a un portafoglio',
+      MAX_LENGTH: 'Il nome deve avere meno di {0} caratteri.',
+      MIN_LENGTH: 'Il nome deve avere almeno 1 carattere | Il nome deve avere almeno {n} caratteri.'
+    },
+    PASSWORD: {
+      TOO_SHORT: 'La tua password deve essere lunga lameno {0} caratteri',
+      NUMBERS: 'La tua password deve contenere almeno 1 numero',
+      SPECIAL_CHARACTERS: 'La tua password deve contenere almeno 1 carattere speciale',
+      NO_MATCH: 'La tua password non coincide'
+    },
+    ADDRESS: {
+      EXISTS_AS_CONTACT: 'L\'indirizzo \'{0}\' è già stato aggiunto come contatto',
+      EXISTS_AS_WALLET: 'L\'indirizzo \'{0}\'è già stato importato come portafoglio'
+    },
+    PUBLIC_KEY: {
+      INVALID_LENGTH: 'La chiave pubblica deve avere una lunghezza di 66 caratteri'
+    },
+    VENDOR_FIELD: {
+      LIMIT_REACHED: 'È possibile inserire un massimo di {0} caratteri'
+    },
+    REQUIRED: 'The \'{0}\' is required',
+    SEND_NOT_ENABLED: 'L\'invio non è abilitato per il portafoglio selezionato',
+    WALLET_NOT_ACTIVE: 'Seleziona un portafoglio a apri l\'URI nuovamente',
+    WARNING_BIP39: '\'{0}\' è una frase segreta BIP39',
+    WARNING_NOT_BIP39: '\'{0}\' non è una frase segreta BIP39'
+  },
+
+  LANGUAGES: {
+    'en-US': 'English',
+    'es-ES': 'Spanish',
+    'it-IT': 'Italian',
+    'pt-BR': 'Portugues - Brazil'
+  },
+
+  TIME_FORMAT: {
+    'DEFAULT': 'Predefinito',
+    '12H': '12h',
+    '24H': '24h'
+  },
+
+  BIP39_LANGUAGES: {
+    chinese_simplified: 'Chinese Simplified',
+    chinese_traditional: 'Chinese Traditional',
+    english: 'Inglese',
+    french: 'Francese',
+    italian: 'Italiano',
+    japanese: 'Giapponese',
+    korean: 'Coreano',
+    spanish: 'Spagnolo'
+  },
+
+  MARKET: {
+    MARKET: 'Mercato',
+    HISTORICAL_DATA: 'Dati storici',
+    DAY: 'Giorno',
+    WEEK: 'Settimana',
+    MONTH: 'Mese'
+  },
+
+  TABLE: {
+    PAGE: 'Pagina',
+    ROWS_PER_PAGE: 'Righe per pagina',
+    NO_TRANSACTIONS: 'Nessuna transazione è stata trovata. Le ultime transazioni verranno visualizzate qui.',
+    NO_CONTACTS: 'Nessun contatto da mostrare.',
+    NO_DELEGATES: 'Nessun delegato da mostrare',
+    NO_WALLETS: 'Nessun portafoglio da mostrare.'
+  },
+
+  APP: {
+    RELEASE: {
+      REQUEST_ERROR: 'Impossibile verificare l\'ultima versione'
+    }
+  },
+
+  APP_FOOTER: {
+    TEXT: 'Made with ♥ by Ark'

```

Comment:
This is missing

Reply:
I left it on purpose in English because it sounds better and is easy to understand even in Italian.
In case you want to change it becomes:

TEXT: 'Made with ♥ by Ark'